index.html

```diff
@@ -16,48 +18,69 @@
 
 <body>
 
- <!-- navigation code here -->
+  <!-- navigation code here -->
+
+  <div class="top">
+    <div class="top-content">
+      <h1>SpaceWalkers</h1>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+        <a href="#">Sign IN</a>
+      </nav>
+    </div>
+  </div>
 
   <div class="container home">
 
     <header class="intro">
-       <!-- image code here -->
+      <!-- image code here -->
+      <!-- <img src="img\main-header.png" class="earth" alt="earth from space"> -->
+      <div class="banner"></div>
 
       <h2>Lift Off</h2>
-    
-      <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
+
+      <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a
+        marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate,
+        that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
     </header>
 
     <section class="space-images">
-        <img src="img/blast.png" class="blast" alt="Rocket blasting off">
-        <img src="img/sky.png" class="sky" alt="Space sky image">
-        <img src="img/launch.png" class="launch" alt="Launching rocket">
+      <img src="img/blast.png" class="blast" alt="Rocket blasting off">
```

Comment:
```suggestion
      <img src="img/blast.png" class="blast" alt="Rocket blasting off" title="<some title that is search engine relevant>">
```
Think about using the title property on img tags to improve SEO.